apps/sms/test/marionette/lib/views/conversation/message_accessors.js

```diff
@@ -52,6 +54,16 @@ MessageAccessor.prototype = {
     }
   },
 
+  selectMessage: function(messageNode) {
+    if (this.parse(messageNode).isSelected) {
+      throw new Error('Message has been selected!');
+    }
+    //messageNode.tap();
+    //messageNode.findElement('li[class=message]').tap();
+    //messageNode.findElement('.pack-checkbox').click();
+    messageNode.findElement('.pack-checkbox').tap();
```

Comment:
I'm unable to successfully select the check boxes. Using inspector, I observed it to be responding whenever the `<label class="pack-checkbox">` is pressed. What could I do @azasypkin ?